options.go

```diff
@@ -0,0 +1,98 @@
+package sendmail
+
+import (
+	"io"
+	"net/mail"
+	"os"
+)
+
+// SetSendmail modifies the path to the sendmail binary. You can pass
+// additional arguments, if you need to.
+func (m *Mail) SetSendmail(path string, args ...string) *Mail {
+	m.sendmailPath = path
+	m.sendmailArgs = args
+	return m
+}
+
+// SetDebug sets the debug output to stderr if active is true, else it
+// removes the debug output. Use SetDebugOutput to set it to something else.
+func (m *Mail) SetDebug(active bool) *Mail {
+	var out io.Writer
+	if active {
+		out = os.Stderr
+	}
+	m.debugOut = out
+	return m
+}
+
+// SetDebugOutput sets the debug output to the given writer. If w is
+// nil, this is equivalent to SetDebug(false).
+func (m *Mail) SetDebugOutput(w io.Writer) *Mail {
+	m.debugOut = w
+	return m
+}
+
+// AppendTo adds a recipient to the Mail. The name argument is the
+// "proper name" of the recipient and may be empty. The address must be
+// in the form "user@domain".
+func (m *Mail) AppendTo(name, address string) *Mail {
```

Comment:
Here we can use `*mail.Address` directly, it's a build in type. I can imagine situation when somebody used `ParseAddress`, which returns `*mail.Address`, then he need to convert it to 2 strings and we convert it back to `*mail.Address`. That would be a bit over engineered. Using mail.Address as a function parameter doesn't bring that much complexity I think: `AppendTo(&mail.Address{Name: "someone", Address: "some@email"})`. @dmke what do you think?